packages/components/Checkbox/SPEC.md

```diff
@@ -50,6 +50,68 @@ Win32:
 <Checkbox label="Circular Checkbox" shape="circular" />
 ```
 
+Android:
+
+##### Light Mode
+
+---
+
+<br/>
+
+![Checkbox with text on Android example](./assets/Checkbox_example_android.png)
+
+```jsx
+<Checkbox label="Unchecked checkbox (undefined)" />
```

Comment:
why do we have (undefined) here..?

Reply:
similar below for "uncontrolled"

Reply:
Undefined means default here , Win32 tester app has demonstrated in the same.  
Seems the spec example and tester example doesn't match for win32. 
I tried to match it for Android , but i can change it to default if required ?